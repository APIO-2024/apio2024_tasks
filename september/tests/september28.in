txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 0 2 3 4 4 5 7 8
1 9 6 8 5 7 3 4 2
10 1
0 0 2 3 4 4 6 7 8
1 9 5 8 7 4 6 3 2
10 1
0 0 1 3 3 5 5 7 8
6 2 9 7 8 5 3 4 1
10 1
0 1 2 2 3 4 5 6 7
7 9 8 6 4 5 3 2 1
10 1
0 0 1 3 4 5 6 6 8
2 7 9 8 6 5 4 3 1
10 1
0 1 1 2 4 4 5 7 7
6 9 8 3 7 5 4 2 1
10 1
0 0 1 2 4 4 6 7 8
7 5 9 8 4 3 1 6 2
10 1
0 1 1 3 3 5 5 7 7
6 4 8 2 9 5 7 3 1
10 1
0 0 1 3 4 4 5 6 8
9 2 8 6 7 5 4 3 1
10 1
0 1 2 3 3 5 5 7 8
9 4 8 6 7 5 3 2 1
10 1
0 1 2 2 3 5 5 6 8
9 4 7 8 6 5 3 1 2
10 1
0 0 2 3 4 5 6 7 7
8 7 9 1 6 5 4 3 2
10 1
0 1 1 3 3 5 6 6 7
2 4 8 9 7 6 5 3 1
10 1
0 0 1 3 4 5 6 6 7
8 2 9 6 7 5 4 3 1
10 1
0 0 2 3 3 5 5 6 7
1 8 7 9 4 2 5 3 6
10 1
0 0 1 3 4 5 6 7 7
9 8 7 1 4 6 2 5 3
10 1
0 0 2 3 3 4 5 6 8
9 8 6 4 1 7 3 5 2
10 1
0 0 2 3 4 5 6 6 7
8 1 9 7 6 5 4 3 2
10 1
0 0 1 2 4 4 6 6 7
5 7 9 3 1 8 2 6 4
10 1
0 0 1 2 3 4 6 7 8
8 9 5 7 1 3 4 6 2
40 1
0 1 0 1 2 3 4 5 8 7 8 9 10 11 12 13 14 15 17 19 18 21 20 21 24 24 24 25 27 28 30 31 31 31 34 33 35 35 37
6 39 32 36 22 26 38 20 23 37 3 29 16 13 27 34 17 10 19 35 31 14 11 33 4 7 12 21 15 25 30 18 24 28 9 8 1 2 5
40 1
0 1 2 3 3 5 4 6 6 7 10 10 11 11 13 15 15 16 17 17 20 21 20 21 23 24 26 25 28 27 29 30 30 32 34 34 34 36 38
19 9 18 39 35 33 38 8 6 36 14 37 22 34 16 32 5 31 29 12 28 25 23 30 27 24 26 20 21 17 15 11 13 10 7 4 2 3 1
40 1
0 1 0 2 2 4 5 6 6 8 9 9 10 11 13 14 16 15 16 17 18 19 22 21 24 24 25 26 28 27 28 30 32 32 32 33 35 35 36
20 37 39 38 23 29 34 3 31 7 28 5 12 26 22 17 19 36 16 33 35 30 32 14 27 11 9 25 24 18 21 15 13 10 8 6 4 2 1
40 1
0 0 2 3 3 4 4 7 8 8 8 11 10 12 14 14 16 16 16 18 18 20 22 22 24 23 25 27 27 27 30 30 30 33 34 35 34 36 38
19 32 29 39 26 17 13 15 38 1 21 9 10 5 36 37 28 31 23 35 33 34 6 30 27 25 24 22 20 18 16 12 14 11 8 4 7 3 2
40 1
0 1 1 3 4 4 6 7 7 9 9 10 12 11 14 13 14 17 17 18 20 21 22 21 24 25 26 26 26 29 30 29 31 32 33 33 34 36 38
35 2 8 28 39 37 15 27 34 19 38 36 5 16 23 32 22 13 12 10 31 33 30 25 26 29 21 24 17 18 20 11 9 7 14 1 6 4 3
100 1
0 0 2 0 2 4 5 4 6 8 10 9 11 11 14 13 15 16 16 16 19 20 19 21 24 22 26 24 27 28 29 29 29 33 33 35 33 35 35 39 38 38 40 40 43 44 46 45 48 47 49 51 50 53 51 52 56 57 56 57 57 59 60 60 63 63 63 66 68 68 68 70 69 73 74 72 74 74 78 77 77 81 82 81 83 84 84 85 88 88 89 89 90 91 91 95 94 96 96
30 76 67 86 37 23 64 79 55 80 61 72 78 34 17 31 12 9 92 36 75 65 15 3 70 28 98 99 54 6 18 14 58 71 41 84 87 97 93 1 42 32 94 59 53 2 96 62 47 95 50 5 38 7 90 91 68 25 69 57 77 19 49 24 45 43 56 48 73 40 51 46 52 89 60 44 74 85 88 66 63 83 21 81 82 33 22 16 10 4 27 20 39 8 11 26 35 13 29
1000 1
0 1 1 1 0 4 6 3 3 5 10 7 11 8 9 15 13 15 13 17 19 21 22 21 19 20 21 26 24 27 25 29 29 30 33 32 34 37 34 34 36 41 40 40 41 42 42 43 43 49 47 51 52 53 54 55 51 52 53 57 60 59 57 62 64 60 61 64 66 65 69 67 69 70 74 73 73 73 74 79 79 78 78 83 83 84 83 87 85 86 87 88 90 92 91 93 96 96 94 94 100 97 97 101 101 104 103 102 107 109 110 110 107 113 113 111 116 112 118 119 119 119 122 123 121 120 126 126 125 126 130 129 129 133 129 133 136 136 135 135 135 137 139 138 144 141 144 147 147 149 149 151 152 149 153 153 152 152 156 155 157 157 161 162 160 160 162 162 167 167 168 166 169 172 171 175 173 175 175 176 179 176 182 182 181 181 182 182 188 185 187 191 191 193 194 195 192 197 193 197 197 198 202 198 201 204 203 207 207 208 206 208 210 211 213 212 214 212 216 215 215 217 218 223 222 222 224 222 227 229 228 230 227 229 231 231 235 235 237 235 240 240 241 240 241 240 242 242 244 244 247 247 248 253 250 252 255 255 254 254 257 256 257 260 263 264 266 265 268 266 265 266 269 268 271 275 272 272 278 279 276 277 277 283 282 281 281 287 285 284 285 286 291 289 290 292 291 295 295 298 300 299 298 303 302 305 304 302 307 306 305 309 311 311 312 312 313 313 315 318 316 318 318 323 320 325 321 327 326 328 330 327 330 329 333 333 334 333 338 336 337 336 341 338 339 344 346 347 343 344 345 349 349 349 353 352 355 354 356 358 358 361 360 363 360 360 363 363 365 364 365 369 371 373 369 370 375 377 377 375 380 379 377 383 380 380 384 386 387 384 390 391 387 389 393 395 391 393 393 395 398 400 400 403 399 405 403 403 406 409 407 409 410 413 412 412 413 412 417 418 416 418 421 423 423 421 423 425 424 427 429 428 431 429 433 435 434 432 438 437 440 438 442 440 442 442 446 447 444 449 447 451 448 452 452 453 453 457 455 455 459 460 457 460 459 461 461 466 464 465 469 471 468 468 472 475 474 474 474 479 475 476 482 478 481 484 486 482 484 489 486 491 492 490 492 493 492 496 496 499 498 497 499 499 503 505 501 506 503 504 510 506 507 510 510 514 512 516 516 518 515 520 517 519 524 522 526 525 528 526 525 530 529 532 531 535 533 537 534 538 537 537 538 538 539 541 543 546 547 547 547 550 548 548 551 553 551 554 553 556 557 557 560 561 564 560 563 565 568 564 566 567 567 571 571 574 575 572 573 575 578 581 581 580 580 583 582 586 584 585 590 586 587 588 593 595 595 593 594 598 597 597 599 602 600 605 601 603 605 609 607 606 608 608 610 610 616 614 614 615 617 619 620 623 621 622 626 627 624 628 630 630 630 628 630 635 632 637 635 639 638 640 639 638 642 645 642 643 644 649 648 648 650 652 652 652 651 656 657 657 659 658 662 659 663 664 662 667 667 667 666 668 670 671 670 673 676 675 676 676 679 677 680 682 679 683 683 682 683 689 685 687 692 691 690 695 695 695 698 694 696 701 698 701 700 701 706 706 704 704 707 707 708 711 712 713 715 713 714 716 717 721 718 722 719 723 724 727 725 726 726 729 727 729 733 733 736 732 733 736 739 741 741 738 744 740 746 743 745 747 747 749 748 750 753 751 756 757 755 759 759 761 760 758 764 764 762 763 768 764 767 767 770 770 772 771 772 773 773 778 779 779 777 780 784 784 784 784 788 784 789 786 787 792 794 793 795 792 797 795 796 800 801 802 803 805 801 803 805 809 805 808 812 813 810 812 813 813 817 819 815 820 817 820 822 823 821 826 828 825 828 827 832 833 830 830 831 834 833 835 840 839 839 840 844 844 843 842 845 846 846 848 852 853 852 855 855 853 854 854 859 858 859 858 860 861 865 866 864 868 870 871 872 869 869 873 876 875 877 878 875 878 881 881 881 884 882 885 886 888 890 887 889 891 890 890 894 893 893 896 898 896 899 898 900 903 904 906 905 905 910 911 909 908 914 913 911 916 917 916 920 920 919 922 919 921 925 925 923 927 928 927 930 929 934 934 931 935 938 937 939 936 938 939 941 943 941 947 944 947 949 950 949 950 951 953 953 953 958 957 959 959 962 959 962 963 963 965 967 964 969 969 970 971 972 970 974 974 973 979 978 979 980 983 980 985 985 987 986 986 985 989 989 989 990 995 992 997 998
414 879 660 77 132 669 806 570 170 961 611 225 672 641 874 127 981 233 245 807 742 991 511 249 730 942 576 239 766 633 540 72 811 226 907 902 790 357 301 555 196 932 880 38 647 604 918 209 237 12 80 31 960 143 348 856 234 487 993 480 782 368 415 693 44 351 895 702 205 402 382 251 200 954 818 99 139 177 454 769 996 646 324 607 45 331 754 999 579 201 379 394 783 401 862 165 863 655 274 798 945 552 544 926 558 25 98 411 146 81 978 221 332 485 791 977 76 50 955 674 921 735 951 445 681 654 542 933 280 768 14 397 68 58 527 589 63 536 521 591 23 273 562 645 470 436 838 692 340 134 389 108 851 342 456 577 554 178 636 407 994 142 293 814 982 804 481 398 310 219 837 523 883 220 612 968 2 763 362 831 148 995 422 462 159 513 164 930 940 901 154 904 535 35 653 802 82 509 420 857 381 705 443 75 750 765 699 243 912 515 601 173 95 71 697 435 531 855 473 785 709 18 602 892 450 728 850 169 584 998 33 618 49 688 458 158 776 319 190 102 948 720 703 185 150 114 376 569 710 613 661 966 370 477 297 771 976 597 189 367 267 323 188 124 698 199 140 195 629 89 291 184 847 494 687 789 347 372 624 91 997 378 992 141 816 834 345 279 479 760 774 388 841 294 631 426 46 467 308 106 957 800 350 915 731 322 232 634 430 246 145 236 829 386 927 965 463 952 278 374 668 592 131 545 640 642 364 339 924 488 22 373 777 28 684 418 270 241 194 500 117 361 269 517 950 590 686 396 315 877 716 483 156 441 385 678 788 306 300 799 665 48 465 984 502 115 682 419 317 989 867 363 596 296 865 936 37 675 174 259 990 568 439 392 26 734 897 824 8 752 860 314 130 346 449 508 574 946 943 482 408 573 975 238 623 849 972 650 836 172 335 404 105 550 704 167 268 216 700 671 677 606 180 749 433 371 843 922 85 988 186 974 507 128 512 116 797 104 417 663 694 20 292 391 666 625 43 403 183 745 476 359 621 828 549 390 258 898 649 826 971 137 497 101 17 289 262 595 40 366 664 336 876 845 775 585 644 431 565 947 285 781 559 400 123 894 444 163 737 495 39 286 440 113 34 256 937 983 617 887 795 914 969 181 135 79 475 931 472 74 7 344 111 726 891 659 471 380 70 56 413 620 882 580 100 427 615 928 356 437 657 355 794 126 779 166 490 352 282 252 215 873 395 651 846 638 691 94 956 65 844 353 925 792 920 637 193 6 247 161 457 120 744 564 448 453 878 685 489 261 16 55 338 916 913 908 160 469 464 679 179 723 230 786 30 632 575 571 54 986 67 823 27 434 906 903 909 460 506 288 287 229 566 872 616 122 539 987 899 227 429 889 893 718 534 501 281 985 772 941 424 64 155 561 224 980 713 886 276 153 62 708 979 840 610 532 223 871 4 884 505 503 498 213 609 605 218 15 410 967 210 738 59 732 530 767 526 212 53 762 52 835 9 729 725 719 830 557 275 208 271 522 61 963 761 333 714 520 207 970 825 973 551 600 3 711 203 330 870 759 328 547 598 265 202 593 962 518 327 587 92 868 543 866 1 88 538 964 87 321 263 257 582 316 755 822 312 255 861 820 309 819 817 250 813 307 859 854 244 959 753 303 958 304 298 812 748 953 949 808 944 939 938 743 929 935 934 923 739 741 919 917 911 910 905 900 896 890 885 888 875 864 881 869 853 858 852 848 842 839 827 832 833 815 821 810 809 805 801 803 796 793 787 780 784 778 773 770 764 758 751 757 756 746 747 740 733 736 727 724 721 722 717 715 712 707 706 701 696 690 695 680 683 689 670 673 676 662 667 652 648 656 658 639 643 635 628 630 627 626 622 619 614 599 608 603 586 594 588 583 578 581 572 567 563 560 556 546 553 548 541 537 533 525 529 528 524 516 514 519 510 504 496 493 492 499 486 491 478 484 474 461 466 468 459 455 451 452 447 446 438 442 428 432 425 423 412 416 421 405 409 406 387 393 399 384 383 375 377 365 369 358 360 349 354 343 341 329 334 337 326 320 325 313 318 311 305 302 299 290 284 295 283 277 264 266 272 260 254 253 248 242 240 231 235 228 222 214 217 211 204 206 198 197 192 191 187 182 171 176 175 162 157 168 151 152 144 138 147 149 136 133 119 121 129 125 118 112 109 110 97 103 107 96 93 90 86 83 84 78 66 69 73 51 60 57 47 42 41 29 36 32 24 21 19 13 11 10 5
100 1
0 0 0 3 4 3 6 4 7 6 9 11 12 11 12 14 16 16 17 18 20 19 19 20 23 23 25 27 28 26 28 29 31 32 33 32 35 35 35 38 38 38 40 42 44 43 44 44 45 47 49 49 50 51 54 53 56 54 56 56 58 60 59 63 63 63 65 64 67 69 69 71 72 72 71 74 73 75 76 77 77 79 81 82 83 84 86 85 88 88 89 88 90 93 93 92 94 94 97
30 22 36 37 62 46 68 13 60 39 57 1 21 8 34 10 80 55 26 87 66 61 78 58 52 48 96 2 32 98 29 43 64 92 15 54 12 75 95 99 70 91 5 86 24 89 49 51 45 41 97 40 20 84 18 94 4 93 90 82 79 88 85 76 83 74 81 77 73 72 71 67 69 65 63 59 56 53 47 50 44 42 38 35 33 31 28 25 27 23 19 16 17 14 11 9 7 6 3
40 1
0 1 2 2 3 4 5 5 8 8 10 11 12 13 12 13 16 17 17 17 19 19 22 23 23 23 25 27 27 28 28 30 31 33 34 33 35 35 36
9 20 24 38 39 7 36 15 18 32 37 26 34 21 35 4 14 6 29 31 27 25 33 28 30 22 23 17 16 19 13 8 12 11 5 1 3 10 2
40 1
0 1 2 1 4 4 5 5 8 9 8 9 11 11 12 15 15 15 17 18 18 21 20 22 22 25 25 25 28 29 28 29 32 31 34 33 34 37 37
5 15 22 3 33 31 21 19 12 14 34 11 28 35 18 16 24 10 9 13 7 37 38 39 6 20 29 26 4 27 17 30 1 2 32 36 8 23 25
40 1
0 1 1 3 3 4 4 6 8 8 8 10 10 12 14 14 15 17 17 18 19 21 22 23 22 23 25 25 26 28 29 31 32 33 32 34 36 37 38
20 2 13 24 11 7 9 35 39 38 16 30 5 27 28 25 37 36 18 33 34 32 31 29 26 23 22 21 17 19 15 14 8 12 10 6 4 3 1
40 1
0 1 0 1 4 5 4 7 6 7 8 9 11 12 14 14 16 16 16 19 19 20 20 22 24 24 24 27 26 29 30 30 32 33 32 35 36 35 37
13 34 8 18 28 11 10 25 21 38 31 23 7 15 33 17 3 2 39 37 36 32 27 35 30 29 22 20 26 24 19 16 14 6 12 9 5 4 1
40 1
0 0 1 2 2 4 5 7 6 9 8 10 12 12 14 15 14 16 16 19 19 19 20 22 22 23 24 25 28 28 29 31 30 33 32 33 36 37 38
18 35 3 11 26 27 23 20 17 8 7 5 13 1 38 24 39 21 29 32 34 31 37 36 25 33 22 28 30 19 14 15 16 12 6 10 9 2 4
10 1
0 1 2 3 3 5 6 6 7
5 1 6 8 4 3 2 7 9
10 1
0 0 1 2 3 4 5 7 8
6 2 9 4 8 7 5 3 1
10 1
0 0 1 2 3 4 6 6 8
5 3 9 7 8 6 1 4 2
10 1
0 1 2 2 3 4 5 7 8
6 3 7 4 5 8 9 2 1
10 1
0 1 1 2 4 4 5 6 8
7 3 9 5 6 8 2 1 4
10 1
0 1 2 2 3 4 6 6 7
8 7 9 5 6 4 3 2 1
10 1
0 1 2 3 3 5 5 6 8
7 9 4 8 6 5 3 2 1
10 1
0 1 2 3 4 5 5 7 8
6 9 8 7 5 4 3 2 1
10 1
0 1 1 3 4 4 5 7 7
8 2 7 9 5 6 4 1 3
10 1
0 1 2 2 4 4 6 7 8
9 8 5 7 3 2 1 4 6
10 1
0 1 2 3 4 4 5 7 8
6 9 8 4 7 5 3 1 2
10 1
0 0 1 2 4 4 5 7 8
3 1 7 9 8 6 2 4 5
10 1
0 1 2 2 3 5 5 7 8
4 6 8 9 5 3 7 2 1
10 1
0 0 1 3 3 4 6 7 8
9 8 2 7 4 6 5 3 1
10 1
0 1 1 3 4 5 6 7 7
2 8 9 7 6 5 1 4 3
10 1
0 0 1 2 3 5 5 7 7
8 4 6 9 2 1 7 3 5
10 1
0 1 1 3 3 4 5 6 7
2 6 8 7 5 9 1 3 4
10 1
0 0 2 3 3 5 6 6 7
1 6 7 5 2 8 9 4 3
10 1
0 1 1 2 4 4 6 6 8
3 5 9 7 8 6 4 1 2
10 1
0 0 2 3 4 4 5 6 7
9 1 8 6 5 7 4 3 2
