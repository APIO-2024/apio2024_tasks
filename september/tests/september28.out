7ckgnn4wyi495puj3ibqf81dqvapyv6b
7
8
7
8
9
9
3
8
9
9
8
8
9
8
5
4
8
9
6
6
21
35
37
36
27
47
900
94
20
1
35
29
25
1
7
9
4
6
8
9
9
7
6
6
5
6
8
7
6
4
2
8
8
